dashboard/app/views/home/_privacy.md

```diff
@@ -187,35 +189,43 @@ Whenever we share Student personal data, we hold our partners to privacy and sec
+Part of our educational program is to offer professional learning workshops to prepare teachers to offer computer science courses. If you are a Teacher participating in one of these workshops, your name and contact information will be shared with the facilitator and/or the local Code.org Regional Partner who runs the professional learning program in your area. In addition, the facilitator, Regional Partner, and your school district will have the opportunity to access your continued progress in our in-person and online professional learning courses in order to coach you, facilitate your additional learning, and follow your professional development progress. (In some cases, your school district may require this information in order to compensate you for workshop attendance). They may also be able to see your overall class progress to support your classroom. The Regional Partner and facilitator will not have any data on specific Students, but they will be able to see the overall number of students and class demographics. If Code.org is paying for your travel to our professional learning workshops for Teachers, we will - with your explicit permission - share your name and contact information with our travel partner to facilitate booking your travel. 
+
+We may also share the list of schools (solely the school identities, without any personal or identifying information about teachers or students) that use Code.org or have participated in our professional learning program with select partners, such as a local government department of education, so long as such partners agree to treat the information as confidential. 
 
-We also provide teachers other opportunities (such as when adding their school to the map of schools teaching Computer Science) to share their contact information (name, school, email) — at their option and under their control — with a Code.org Regional Partner so that teachers can be contacted about local professional learning workshops, resources, and events. 
+We also provide teachers other opportunities (such as when adding their school to the map of schools teaching Computer Science) to share their contact information (name, school, email) — at their option and under their control — with a Code.org Regional Partner so that teachers can be contacted about local professional learning workshops, resources, and events.
```

Comment:
Can we link to /yourschool here?

Reply:
Any particular reason you think we should? We aren't linking to other forms we refer to.

Reply:
I just had the thought while reading this, "what map?" I don't think it's necessary. Just helpful.